app/utils/auth_middleware.py

```
@@ -0,0 +1,25 @@
+from flask_jwt_extended import verify_jwt_in_request, get_jwt
+from functools import wraps
+from flask import request, jsonify, redirect, url_for
```

Ruff (0.8.2)

`flask.request` imported but unused

Remove unused import

(F401)